terraform/modules/compute/main.tf

```
@@ -0,0 +1,77 @@
+############################
+# EC2 Instance
+############################
+resource "aws_instance" "ec2" {
+  ami                    = var.ami_id
+  instance_type          = var.instance_type
+  key_name               = var.key_name
+  subnet_id              = var.public_subnet_ids[0]
+  vpc_security_group_ids = [var.ec2_sg_id]
+
+tags = {
+    Name        = "${var.prefix}-ec2"
+    Environment = "lab"
+    Project     = var.prefix
+  }
```

```suggestion
  tags = {
    Name        = "${var.prefix}-ec2"
    Environment = "lab"
    Project     = var.prefix
  }
```
インデントがズレています！
VS CodeのTerraform用の拡張機能は導入済みでしょうか？こちらが導入済みであれば、保存時に自動的にフォーマットできるはずです！

---

ありがとうございます。設定のformat on saveが有効でなかったことが原因だったかなと思います。